app/components/sections/card_post/SectionCardPost.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { SectionAboutPostProps } from "../../../constants/types/components_props/types"
+import Link from "../../link/Link"
+import SectionAboutPost from "../about_post/SectionAboutPost"
+
+interface SectionCardPostProps extends SectionAboutPostProps {
```

Comment:
could we add this and other interfaces to a separate file and then import them where required?
For organization (example as types.ts file)